test_neg_view: preseve sign of sample input

peterbell10 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #63011
* __->__ #63010

This changes `test_neg_view` to call the operator with the same numeric values as the original sample input.

Differential Revision: [D31082824](https://our.internmc.facebook.com/intern/diff/D31082824)

peterbell10 committed:
test_neg_view: preseve sign of sample input

This changes `test_neg_view` to call the operator with the same numeric values as the original sample input.

[ghstack-poisoned]

anjali411 commented on test/test_ops.py:
```diff
@@ -972,10 +972,10 @@ def test_conj_view(self, device, dtype, op):
         if not op.test_neg_view:
             self.skipTest("Operation not tested with tensors with negative bit.")
-        math_op_physical = torch.neg
+        math_op_physical = torch.clone
```
can you leave a comment here explaining why we have clone here instead of neg (which is expected given the name of the var `math_op_physical`)

anjali411 replied:
inserting the PR description here would be sufficient too

anjali411 reviewed (approved):
LGTM thank you!

peterbell10 committed:
Update on "test_neg_view: preseve sign of sample input"


This changes `test_neg_view` to call the operator with the same numeric values as the original sample input.

[ghstack-poisoned]

peterbell10 commented:
@anjali411 I just noticed this was never landed.

peterbell10 committed:
Update on "test_neg_view: preseve sign of sample input"


This changes `test_neg_view` to call the operator with the same numeric values as the original sample input.

Differential Revision: [D31082824](https://our.internmc.facebook.com/intern/diff/D31082824)

[ghstack-poisoned]

peterbell10 committed:
Update on "test_neg_view: preseve sign of sample input"


This changes `test_neg_view` to call the operator with the same numeric values as the original sample input.

Differential Revision: [D31082824](https://our.internmc.facebook.com/intern/diff/D31082824)

[ghstack-poisoned]

anjali411 commented:
@anjali411 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31082824).
